docs/updates/infromational-dec-26-2021.md

```diff
@@ -0,0 +1,89 @@
+    gravity104vmnec7p96qgzzdzddv5jan070v7vj6j7mslj,
+    gravity19lhajns3drveve4q8jc4kus6knjcmf2j36ta6e,
+} and bugged validators 11 {
+    gravityvaloper19lhajns3drveve4q8jc4kus6knjcmf2jq3jrsd,
+    gravityvaloper12qx44c75dd8ykckexxxxjn7ygehwxkqjh04uke,
+    gravityvaloper19uwnjc4fhftygyzr8wv5m8m0l4kjzkpsamzk2r,
+    gravityvaloper104vmnec7p96qgzzdzddv5jan070v7vj6r4zw4x,
+    gravityvaloper1gfeh4793ug8l6nu69n8lqssvkdqs8qghld0r9t,
+    gravityvaloper1kp6kn7jazzn6leqvqzdm9ftmlpp72l40tz9dqh,
+    gravityvaloper1vu04rhxr4sq65vfd074m5gjzuggwukzhncmnzk,
+    gravityvaloper1qtc3axqj2pf9h92arc2lmk7keyvrzqc847w8r9,
+    gravityvaloper1xmspr43xsfu6ycjgdqqllntc62rzvncv0d74fz,
+    gravityvaloper1xf869399xq4pnxsllzvnwdcra8ly3huj5cnwh5,
+    gravityvaloper1d63hvdgwy64sfex2kyujd0xyfhmu7r7cgxt5ru,
+} total affected stake 1803249988137
+
+```
+
+As you can see all affected delegations are self-delegations to genesis validators. Each holding `181,818 graviton` with slight
+variations based on withdraws and redelegation before their slashing and exactly how many times they have been slashed by the Gravity module.
+
+### Recommended Action
+
+The latest release version of Gravity Bridge `v1.1.0` and greater correctly set the hooks in the Gravity module `app.go`. It is important that the community upgrade to `v1.1.0` or later release or any validator that is slashed by the Gravity Bridge module will be unable to withdraw delegation rewards.
+
+Although unbonding of the delegated tokens will function correctly all delegators to that validator, which includes the validator themselves, will be unable to withdraw rewards after the slashing occurs.
+
+### Resolution for affected individuals
+
+The most correct way to resolve this issue is to insert the correct slashing events into the history during a chain upgrade. This would allow the affected validators to withdraw their rewards with no disruption.
+
+Sadly this option has significant downsides. Since it can only be done during chain upgrade the upgrade to `v1.1.0` can either be delayed. Risking more validators encountering the issue, or the affected users may have to wait until the following upgrade to be able to withdraw their rewards.
+
+Furthermore given the compounding effect of withdrawing and re-delegating simply allowing affected individuals to withdraw rewards does not compensate for opportunity cost.
+
+The community may choose to use a Gravity Bridge [Airdrop Proposal](https://github.com/Gravity-Bridge/Gravity-Bridge/blob/main/module/proto/gravity/v1/types.proto#L67) to send users who have experienced this issue a flat amount that can be computed to include opportunity cost.
+
+### Recommendation for affected individuals
+
+Regardless of how the community choses to resolve this issue the fastest way to get back on track staking for an affected individual is to begin unbonding and re-delegate their tokens to a new validator, unaffected by the bug. Once the unbonding period ends.
```

Comment:
Again I'm fairly certain that an affected validator will not be able to move their delegation around or undelegate based on the staking and distribution code I have looked at. Were you able to test this?

Reply:
testing it locally now. Will get back to you.

Reply:
haven't really been able to test conclusively. My laptop isn't really fast enough to get good data here. Going to go ahead with the assumption that unbonding does not work.